actors/builtin/miner/partition_state_test.go

```diff
@@ -58,15 +58,22 @@ func TestPartitions(t *testing.T) {
 		})
 	})
 
+	t.Run("doesn't add sectors twice", func(t *testing.T) {
+		_, store, partition := setup(t)
+
+		_, err := partition.AddSectors(store, sectors[:1], sectorSize, quantSpec)
+		require.Error(t, err)
```

Comment:
I like to add an `assert.Contains(t, err.Error(), "...") after this to ensure it's failing for the right reason. I've found a lot of existing error tests that were not.